src/idl/src/scope.c

```diff
@@ -38,6 +52,7 @@ create_declaration(
     goto err_identifier;
   if (idl_create_name(pstate, idl_location(name), identifier, &declaration->name))
     goto err_name;
+  declaration->name->is_annotation = name->is_annotation;
```

Comment:
I have only one comment: would it not be better to let `idl_create_name` set the `is_annotation` field based on a parameter?

Reply:
Yes, I also considered that, but the first try didn't work out well (because the call from generic `identifier` in the parser), so I decided not to do that. But looking at this again, I realized that based on the parser state, I can set the value for that flag at that point. I've changed this in e402ec2